### ISSUE TITLE
Using Router in Frontend

### ISSUE THREAD

**Issue by CahidArda** [bounswe/bounswe2022group7#268]
We need to update the frontend app using router. Router shall direct the user to pages with static and dynamic routes. An example to the static route is the home page. An example to the dynamic page is the page of an art item. The [URL parameters sandbox of React Router](https://v5.reactrouter.com/web/example/url-params) can be used as reference when implementing the dynamic routes.

To lay the foundations, I will add the pages I provided as examples above: home page and the art item page. Home page already exists as implemented by @guneyizol. I will add a more or less empty art item page to test the Router.

Reviewer: @guneyizol 
Deadline: 20.10.2022

**Comment by CahidArda** [comment]
I have completed the router implementation. I have done the following changes:
- Change `react-router` package with `react-router-dom` because I was more familier with it. I also saw that there is more functionality in `react-router-dom` ([ref](https://stackoverflow.com/questions/42684809/react-router-vs-react-router-dom-when-to-use-one-or-the-other))
- Create a pages folder to hold each page in the app and related files. Move home-page-related files to a new folder `src/pages/HomePage`
- Create `App.js` and add router. Call `App` from `index.js`
- Implement the routing of the art item pages to demonstrate dynamic routing.

I will now create an PR to merge my changes into the front-end dev branch. This issue can be closed once the MR is closed.
